frontend/src/StartView.svelte

```diff
@@ -134,15 +136,33 @@
       showRootResource = false;
       return;
     }
-
-    const rootModel = await fetch(
-      `${$settings.backendUrl}/create_root_resource?name=${f.name}`,
-      {
-        method: "POST",
-        body: await f.arrayBuffer(),
+    if (f.size > warnFileSize) {
+      for (var start = 0; start < f.size; start += fileChunkSize) {
```

Comment:
```suggestion
      for (let start = 0; start < f.size; start += fileChunkSize) {
```

Do not use `var`!

- <https://medium.com/@codingsam/awesome-javascript-no-more-var-working-title-999428999994>
- <https://codeburst.io/javascript-var-let-or-const-which-one-should-you-use-2fd521b050fa>

Reply:
woops! this should be outdated now anyway